scripts/wca/auro-flight.js

```diff
@@ -0,0 +1,28 @@
+
+import { AuroFlight } from '../../src/auro-flight.js';
+
+/**
+ * The auro-flight element renders a DoT compliant Flight listing.
+ * This design has been tested via the Alaska Legal team for legal compliance.
+ * Please DO NOT modify unit tests pertaining to DoT regulations.
+ *
```

Comment:
I think this is a problem here. The description for the component is in `../../src/auro-flight.js` and I don't think moving it to here (or keeping two instances of it is a good idea. 

Is there another way we can resolve this, have the API build with the description and not have to include it here in this wca script?